src/components/pages/page/common/sidebar/pages-list.tsx

```diff
@@ -43,6 +47,15 @@ const PagesListItem = ({ page, isParent }) => {
             {page.title}
           </chakra.a>
         </Link>
+        {canEdit && page.pageType == "Redirect" && (
```

Comment:
can we have pageTypes in constants file and then access that here instead of hardcoding?